heidegger_index/templates/work_detail.html

```diff
@@ -12,10 +12,32 @@ <h2>Index terms</h2>
 {% regroup page_refs by lemma as lemma_list %}
 <ul class="lemma-refs">
   {% for lemma, page_ref_list in lemma_list %}
-    <li>
-      <a href="{% url 'index:lemma-detail' lemma.slug %}" class="hidden-link">{{ lemma.format }}</a>: {{ page_ref_list|join:', ' }}
-    </li>
+    {% if not lemma.type %}
+      <li>
+        <a href="{% url 'index:lemma-detail' lemma.slug %}" class="hidden-link">{{ lemma.format_just_lemma }}</a>: {{ page_ref_list|join:', ' }}
+      </li>
+    {% endif %}
+  {% endfor %}
+</ul>
+<h2>People mentioned</h2>
+<ul class="lemma-refs">
+  {% for lemma, page_ref_list in lemma_list %}
+    {% if lemma.type == 'p' %}
+      <li>
+        <a href="{% url 'index:lemma-detail' lemma.slug %}" class="hidden-link">{{ lemma.format_just_lemma }}</a>: {{ page_ref_list|join:', ' }}
+      </li>
+    {% endif %}
+  {% endfor %}
+</ul>
```

Comment:
Wat je hier (en op een vergelijkbare manier bij de werklemma's) denk ik het beste kunt doen is in `WorkDetailView.get_context_data` een query maken die de persoonslemma's filtert en die meegeven aan de context, bijv.:

```python
context['person_list'] = Lemma.objects.filter(work=context["work"], type='p')
```

Dan kan je om bovenstaand stuk in de template een simpele `{% if person_list %}` zetten om er voor te zorgen dat deze HTML-snippet verdwijnt wanneer er geen persoonslemma's zijn. Bovendien voorkom je zo dat je drie keer over de hele `lemma_list` moet itereren om het template te renderen.

Reply:
Goede oplossing. Ik zat er al mee te puzzelen!

Reply:
Ik heb dit toegepast in [deze commit](https://github.com/andredelft/heidegger-index/pull/7/commits/0de47e5b81191953bc16c362cf6987839d1aef83).